configs/eslint-config-typescript-react/index.js

```diff
@@ -0,0 +1,41 @@
+const OFF = 0;
+const WARN = 1;
+const ERROR = 2;
+
+module.exports = {
+  root: true,
+  extends: [
+    '@monkvision/eslint-config-typescript',
+  ],
+  plugins: ['jest', '@typescript-eslint', 'import', 'prettier'],
```

Comment:
do you really need to redefine all of those plugins here? Since they are already defined in @monkvision/eslint-config-typescript

Reply:
Done